packages/flutter/lib/src/widgets/text_selection.dart

```diff
@@ -207,7 +207,6 @@ abstract class TextSelectionControls {
   /// by the user.
   void handleSelectAll(TextSelectionDelegate delegate) {
     delegate.selectAll(SelectionChangedCause.toolbar);
-    delegate.bringIntoView(delegate.textEditingValue.selection.extent);
```

Comment:
I believe this is redundant: it seems like the default `editableTextState.selectAll` calls scrollIntoView

Reply:
Looks like you're right and it should be removed.  I'll be deprecating handleSelectAll soon and hopefully removing the opportunity for things like this to happen.